app/upgrades/v0_4_1/upgrades.go

```diff
@@ -0,0 +1,27 @@
+package v0_4_1 //nolint:revive // var-naming: don't use an underscore in package name
+
+import (
+	"context"
+
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+	"github.com/allora-network/allora-chain/app/upgrades"
+	"github.com/cosmos/cosmos-sdk/types/module"
+)
+
+const (
+	UpgradeName = "v0.4.1"
```

Comment:
needs to be v0.5.0. 

The folder also needs to be renamed.

Reply:
https://github.com/allora-network/allora-chain/pull/594